nexus_demos/src/rmf_task_canceller.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2025 Open Source Robotics Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SRC__RMF_TASK_CANCELLER_HPP_
```

Comment:
though currently it is highly unlikely to cause a header guard collision, we can consider keeping to the format of `NEXUS_DEMOS__SRC__RMF_TASK_CANCELLER_HPP`